subsys/bluetooth/controller/Kconfig

```diff
@@ -470,20 +470,17 @@ config BT_CTLR_CHAN_SEL_2
 	  the Controller.
 
 config BT_CTLR_ADV_EXT
-	bool "LE Advertising Extensions" if !BT_LL_SW_SPLIT
+	bool "LE Advertising Extensions"
 	depends on BT_CTLR_ADV_EXT_SUPPORT
 	select BT_CTLR_CHAN_SEL_2 if BT_LL_SW_SPLIT && BT_BROADCASTER
 	select BT_CTLR_SCAN_REQ_NOTIFY if BT_LL_SW_SPLIT && BT_BROADCASTER
+	select EXPERIMENTAL if BT_LL_SW_SPLIT
 	# Enable by default for BT_LL_SW_SPLIT when "LE Advertising Set Terminated event" is implemented
 	default y if BT_EXT_ADV && !BT_LL_SW_SPLIT
 	help
 	  Enable support for Bluetooth 5.0 LE Advertising Extensions in the
 	  Controller.
 
-config BT_CTLR_ADV_EXT
-	prompt "LE Advertising Extensions [EXPERIMENTAL]" if BT_LL_SW_SPLIT
-	select EXPERIMENTAL if BT_LL_SW_SPLIT
-
```

Comment:
These changes are intentional:
- One declaration of `BT_CTLR_ADV_EXT` instead of two
- Move `select EXPERIMENTAL if BT_LL_SW_SPLIT` to the only declaration remaining. 

This makes it appear nicer than what is today (see attached screenshot)

The other alternative is to add `depends on BT_LL_SW_SPLIT` to the second declaration, but that would only solve the confusion partly. There would still be two entries turning up when searching.